ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -523,6 +523,8 @@ protected Boolean findNextCompactionAndExecute(boolean computeStats) throws Inte
       final StatsUpdater su = computeStats ? StatsUpdater.init(ci, msc.findColumnsWithStats(
           CompactionInfo.compactionInfoToStruct(ci)), conf,
           runJobAsSelf(ci.runAs) ? ci.runAs : t.getOwner()) : null;
+      Path resultDir = QueryCompactor.Util.getCompactionResultDir(sd, tblValidWriteIds,
```

Comment:
make it final.

Reply:
Add a comment - that this is the path which the compactor is supposed to write to, including an example path + the fact that it has a _v<txn> in it